library/core/src/main/java/com/google/android/exoplayer2/text/subrip/SubripDecoder.java

```diff
@@ -95,8 +128,36 @@ protected SubripSubtitle decode(byte[] bytes, int length, boolean reset) {
         textBuilder.append(currentLine.trim());
       }
 
-      Spanned text = Html.fromHtml(textBuilder.toString());
-      cues.add(new Cue(text));
+      // Extract tags
+      SubtitleTagResult tagResult = extractTags(textBuilder);
+      Spanned text = Html.fromHtml(tagResult.cue);
+
+      Cue cue = null;
+
+      // Check if tags are present
+      if (tagResult.tags.length > 0) {
```

Comment:
nit - You can probably remove this. The loop will be a no-op if tags has length zero.